pkg/reconciler/v1alpha1/clusteringress/clusteringress_test.go

```diff
@@ -94,14 +94,17 @@ func TestReconcile(t *testing.T) {
 						},
 					},
 					Conditions: duckv1alpha1.Conditions{{
-						Type:   v1alpha1.ClusterIngressConditionLoadBalancerReady,
-						Status: corev1.ConditionTrue,
+						Type:     v1alpha1.ClusterIngressConditionLoadBalancerReady,
+						Status:   corev1.ConditionTrue,
+						Severity: "Error",
```

Comment:
To avoid modifying things everywhere, can we have a Conditions wrapper in test, that default the `Severity` to "Error" if not specified?

Reply:
I debated adding `Severity` to the set of things ignored in the base table test (currently: `LastTransitionTime`), but we still need to provide this to our inputs or we'll get hard to decode "unexpected update" errors that seemingly match what we passed in.

It is clear that we need to refactor these table tests so there is less churn, but I don't want to block `knative/pkg` updates waiting on that.  I'll play with this tomorrow.

Reply:
> but I don't want to block knative/pkg updates waiting on that. I'll play with this tomorrow.

I'm fine with this